src/Symfony/Component/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -29,6 +31,27 @@ class MarkdownDescriptor extends Descriptor
     /**
      * {@inheritdoc}
      */
+    public function describe(OutputInterface $output, $object, array $options = array())
+    {
+        $decorated = $output->isDecorated();
+        $output->setDecorated(false);
+
+        parent::describe($output, $object, $options);
+
+        $output->setDecorated($decorated);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function write($content, $decorated = true)
```

Comment:
Don't know if it's something acceptable to change the default value of an argument (`$decorated`) in a method for such case.